gbpservice/nfp/Automation-Scripts/NFP_SCRIPTS/vyos_fw.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+source /home/stack/devstack/openrc neutron service
+
+#service chain node and spec creation
+gbp servicechain-node-create  --service-profile vyos_fw_profile --template-file ./templates/fw_template.yml FWNODE
+gbp servicechain-spec-create --nodes "FWNODE" fw-chainspec
```

Comment:
Should we also rename the service chain node so that it reflects the vyos name? When both vyos and asav firewalls are created, it will be confusing I think.